build.rs

```diff
@@ -28,6 +28,9 @@ fn build_afl(out_dir: &Path) {
         .env("NO_BUILD", "1")
         .env("DESTDIR", out_dir)
         .env("PREFIX", "");
+    if let Ok("false") = std::env::var("DEBUG").as_deref() {
```

Comment:
```suggestion
    if std::env::var("DEBUG").as_deref() == Ok("false") {
```
Sorry to nit, but I don't think destructuring is necessary here.

Reply:
I think that's my Haskell background showing; let's do it your way.